line/__tests__/functionality_unit.test.mjs

```diff
@@ -0,0 +1,21 @@
+import app from '../../app.mjs';
```

Comment:
You do not need this import.  I had thought we needed it for coverage previously, but after looking into it we don't need this import in the test files unless we use `app` in the test.